### PR TITLE
PWGLF: Update lambda1520SpherocityAnalysis.cxx

### PR DESCRIPTION
**Pull request by yashpatley**

### CODE REVIEW

File: PWGLF/Tasks/Resonances/lambda1520SpherocityAnalysis.cxx [hunk 1]
```diff
@@ -351,12 +351,61 @@ struct lambdaAnalysis {
     return true;
   }
 
+  template <typename T>
+  void fillQAHistos(T const& track)
+  {
+
+    // get total momentum
+    float p = TMath::Sqrt(track.px() * track.px() + track.py() * track.py() + track.pz() * track.pz());
```
Comment (vkucera) [inline PWGLF/Tasks/Resonances/lambda1520SpherocityAnalysis.cxx:359]:
```suggestion
    float p = track.p();
```

Comment (yashpatley) [reply PWGLF/Tasks/Resonances/lambda1520SpherocityAnalysis.cxx:359]: I think it is not available in resodaughter table in LFResonanceTables.h so I can't use it.

Comment (yashpatley) [reply PWGLF/Tasks/Resonances/lambda1520SpherocityAnalysis.cxx:359]: Okay I will do that. Thank you.

**Review by BongHwi — APPROVED** [review]

**Comment by BongHwi** [comment]
@yashpatley , please fix this PR

**Review by BongHwi — DISMISSED** [review]

**Review by BongHwi — APPROVED** [review]

**Pull request merged**